Conflux.API/Controllers/ProjectController.cs

```diff
@@ -66,4 +68,23 @@ public async Task<ActionResult> UpdateProject([FromRoute] Guid id, ProjectUpdate
         Project? updateProject = await _projectService.UpdateProjectAsync(id, projectDto);
         return updateProject == null ? NotFound() : Ok(updateProject);
     }
+
+    /// <summary>
+    /// Updates a project by adding the given person.
+    /// </summary>
+    /// <param name="projectId">The GUID of the project to update</param>
+    /// <param name="personId">The GUID of the person to add to the project</param>
+    /// <returns>The request response</returns>
+    [HttpPost]
+    [Route("{projectId:guid}/addPerson/{personId:guid}")]
+    public ActionResult<Project> AddPersonToProject([FromRoute] Guid projectId, Guid personId)
+    {
+        Project? project = _context.Projects.FirstOrDefault(p => p.Id == projectId);
+        Person? person = _context.People.FirstOrDefault(p => p.Id == personId);
+        if (project == null || person == null) return NotFound();
+        
+        project.People.Add(person);
+        _context.SaveChanges();
+        return GetProjectById(project.Id);
```

Comment:
De logica die hier staat is wel correct maar kan beter. Zie de suggestie hieronder.
```suggestion
        Project? project = _context.Projects
            .Include(p => p.People)
            .Include(p => p.Products)
            .Include(p => p.Parties)
            .FirstOrDefault(p => p.Id == projectId);

        if (project == null) return NotFound("Project not found");

        Person? person = _context.People.FirstOrDefault(p => p.Id == personId);

        if (person == null) return NotFound("Person not found");

        project.People.Add(person);
        _context.SaveChanges();
        return Ok(project);